package.json

```diff
@@ -18,24 +18,31 @@
   },
   "homepage": "https://github.com/pstros/angular-jitsi-meet",
   "dependencies": {
-    "jitsi-meet": "git+https://github.com/pstros/jitsi-meet.git"
+    "jitsi-meet": "git+https://github.com/pstros/jitsi-meet.git#4d7bce8baf29e08a38dc09111cb7f73b909fece9"
```

Comment:
what is the significance of this commit hash?  does it point to the `integration` branch?  if so, can it do so in a way that follows changes to the branch?